experiment_helpers/adjust_experimentSettings.m

```diff
@@ -0,0 +1,134 @@
+function [expt] = adjust_experimentSettings(expt, h_fig, adjustment)
+% This function can be called by pressing 'e' on the keyboard during an experiment ('e' for experiment settings) 
+% 
+% Currently only used by taimComp but theoretically could be used by any experiment that wanted to change any of these
```

Comment:
Can you comment generally what would need to be done to generalize it? eg, "the targetdur input param assumes taimComp structure. You could make a new input arg for single-OST experiments..." etc.